modules/openapi-generator/src/main/resources/Java/libraries/jersey2/model_doc.mustache

```diff
@@ -0,0 +1,19 @@
+{{#models}}{{#model}}
+
+{{#isEnum}}
+{{>enum_outer_doc}}
+{{/isEnum}}
+{{^isEnum}}
+{{^oneOf.isEmpty}}
+{{>model_oneof_doc}}
```

Comment:
What about when properties are set in the composed schema and we have oneOf or anyOf?